Gemfile

```diff
@@ -40,3 +40,30 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'rest-client'
 gem 'rspec-rails'
+
+#
+# Custom Gemfile modifications
+#
+# To develop a gem locally and override its source to a checked out repo
+#   you can use this helper method in Gemfile.dev.rb e.g.
+#
+# override_gem 'manageiq-ui-classic', :path => "../manageiq-ui-classic"
```

Comment:
The comment should probably be updated to use `topological_inventory-core`.

Reply:
:+1: done